src/Symfony/Bundle/SecurityBundle/Templating/Helper/LogoutUrlHelper.php

```diff
@@ -68,7 +72,7 @@ public function registerListener($key, $logoutPath, $csrfTokenId, $csrfParameter
      *
      * @return string The logout path
      */
-    public function getLogoutPath($key)
+    public function getLogoutPath($key = null)
```

Comment:
People extending the class will have to update their method signatures. That's why the [BC promise](http://symfony.com/doc/current/contributing/code/bc.html#changing-classes) requires this change.